source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -243,6 +243,46 @@ number of writes may differ between source and destination.
 Transactions might not appear atomically on the destination cluster.
 Retryable writes may not be retryable on the destination cluster.
 
+.. _c2c-verifier-disclaimer:
+
+Embedded Verifier Disclaimer
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting in 1.9, ``mongosync`` includes an embedded verifier to
+perform a series of verification checks on all supported
+collections on the destination cluster to confirm that
+it was successful in transferring documents from the
+source cluster to the destination.
+
+When you start the ``mongosync`` process, it provides a
+disclaimer advising the user that the verifier is enabled
+by default.
+
+.. code-block:: none
+
+   Embedded verification is enabled by default for replica set to replica set migrations.
```

Comment:
Sorry for the late notice, this disclaimer text was tweaked a little bit recently. It is here: https://github.com/10gen/mongosync/blob/da14869ea9a5a1b6593eb0e783543b9b9add70ea/internal/mongosync/app_prod.go#L14

Reply:
Fixed